core/src/com/po/fuck/model/constants/BalanceConstants.java

```diff
@@ -0,0 +1,27 @@
+package com.po.fuck.model.constants;
+
+import java.nio.file.Paths;
+import java.util.Properties;
+
+import static com.po.fuck.model.constants.ConstantsLoader.loadProperties;
+import static com.po.fuck.model.constants.ConstantsLoader.loadFloat;
+import static com.po.fuck.model.constants.ConstantsLoader.loadInt;
+
+public final class BalanceConstants extends BaseConstants {
+    static final String BALANCE_PROPERTIES_PATH = Paths.get(PROPERTIES_FOLDER, "/balance.properties").toString();
+    static final Properties BALANCE_PROPERTIES = loadProperties(BALANCE_PROPERTIES_PATH);
```

Comment:
No visibility modifier.

Reply:
It should be visible only for package. Thus, it is correct?